.github/workflows/build-release.yaml

```
@@ -0,0 +1,56 @@
+name: Build and release
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - created
+
+jobs:
+  build-release:
```

This permission should be here in order to enable the provenance feature:
```suggestion
  build-release:
      permissions:
      id-token: write
```